starters/next-wordpress-starter/components/Post.js

```diff
@@ -0,0 +1,37 @@
+import Image from "next/image";
+import Link from "next/link";
+
+export default function Post({
+  post: { title, date, featuredImage, content },
+}) {
+  return (
+    <article className="prose lg:prose-xl mt-10 mx-auto">
+      <h1>{title}</h1>
+      <p className="text-sm text-gray-600">{date}</p>
```

Comment:
One more small suggestion here:
```suggestion
      <p className="text-sm text-gray-600">{new Date(date).toDateString()}</p>
```

Reply:
Oh, thank you! I was thinking about if I need to format it but forgot to ask :p